### PR TITLE
Add OrangePI R1 detection

### PR DESCRIPTION
**Pull request by kouliss**

### CODE REVIEW

File: adafruit_platformdetect/board.py [hunk 1]
```diff
@@ -336,6 +337,14 @@ def _armbian_id(self):
             return ORANGE_PI_PC
         return None
 
+    def _armbian_id(self):
```
Comment (ladyada) [inline adafruit_platformdetect/board.py]: this code should be merged with the above, not replace!

Comment (kouliss) [reply adafruit_platformdetect/board.py]: You mean like this?

def _armbian_id(self):
        """Check whether the current board is an OrangePi PC."""
        board_value = self.detector.get_armbian_release_field('BOARD')
        if board_value == "orangepipc":
            return ORANGE_PI_PC
	if board_value == "orangepi-r1":
            return ORANGE_PI_R1
        return None

Comment (ladyada) [reply adafruit_platformdetect/board.py]: yep

Comment (kouliss) [reply adafruit_platformdetect/board.py]: OK, changed

**Comment by makermelissa** [comment]
I noticed a couple more things that need to be added. 

We'll need an orange IDs array like
```
_ORANGE_PI_IDS = (
    ORANGE_PI_PC,
    ORANGE_PI_R1
)
```

Also we'll need the `any_orange_pi` property function modified to look in the array with something like:
```
    @property
    def any_orange_pi(self):
        """Check whether the current board is any defined Orange Pi."""
        return self.id in _ORANGE_PI_IDS
```

**Comment by kouliss** [comment]
changed and updated

File: adafruit_platformdetect/board.py [hunk 2]
```diff
@@ -25,8 +25,6 @@
 GENERIC_LINUX_PC            = "GENERIC_LINUX_PC"
 PYBOARD                     = "PYBOARD"
 NODEMCU                     = "NODEMCU"
-ORANGE_PI_PC                = "ORANGE_PI_PC"
```
Comment (makermelissa) [inline adafruit_platformdetect/board.py]: You'll want to leave both of these IDs here too. This is where they are defined.

File: adafruit_platformdetect/board.py [hunk 3]
```diff
@@ -384,7 +388,7 @@ def any_beaglebone(self):
     @property
     def any_orange_pi(self):
         """Check whether the current board is any defined Orange Pi."""
-        return self.ORANGE_PI_PC
+        return self.ORANGE_PI_IDS
```
Comment (makermelissa) [inline adafruit_platformdetect/board.py]: This should return a True/False value instead of a list. Also, the list starts with an underscore character.

**Comment by makermelissa** [comment]
At a glance, this looks good. I'll test it.

**Comment by makermelissa** [comment]
@ladyada, looks like you'll need to merge this in. I don't have the permissions currently.

**Comment by ladyada** [comment]
ok cool to verify you got an R1?

**Comment by makermelissa** [comment]
Yes I did.

**Comment by ladyada** [comment]
rad, ill give you perms too

**Comment by makermelissa** [comment]
Thanks :)

**Pull request merged**